qcodes/utils/update/qcodes_updater.py

```diff
@@ -0,0 +1,110 @@
+
+        # Copy QCoDeS root to qcodes_backup
+        _destination = _source + '_backup' + '_' + _time
+        shutil.copytree(_source, _destination, symlinks=True, ignore=None)
+
+        # Re-install QCoDeS from the back up root
+        subprocess.run(f'activate {env_backup_name} && pip uninstall qcodes && pip install -e {_destination}', shell=True)
+
+        return _source, _destination, env_name, env_backup_name
+
+def conda_env_update(conda_update: bool = True, env_update: bool = True):
+    """
+    Update Conda package manager and QCoDeS Anaconda Python environment
+
+    Args:
+        conda_update: Update Conda pakage manager
+        env_update: Update Anaconda Python environment
+    """
+    # Back up first
+    _source, _destination, env_name, env_backup_name = qcodes_backup()
+
+    # Pull QCoDeS master
+    repo = git.Repo(_source)
+    _git = repo.git
+    _git.checkout('master')
+    _git.pull()
+
+    # Update Conda
+    if conda_update:
+        print("Updating Conda...\n")
+        subprocess.run('conda update -n base conda -c defaults', shell=True)
+
+    # Update QCoDeS Anaconda Python environment
+
+    # Note that Conda will not upgrade a package, even if there is a newer
+    # version, in the case that the package satisfies the requirement specified
+    # by QCoDeS.
+
+    if env_update:
+        print("Updating QCoDeS environment...\n")
+        subprocess.run('conda env update', shell=True, cwd=_source)
+
+    return _source, _destination, env_name, env_backup_name
+
+def update_qcodes_installation():
+
+    """
+    Update current QCoDeS Anaconda Python installation.
+
+    Args:
+     qcodes_update: Updates QCoDeS module
+
+    Raises:
+        ImportError: If QCoDeS module cannot be imported after the update
+    """
+    # Update environment first
+    _source, _destination, env_name, env_backup_name = conda_env_update()
+
+    # Update QCoDeS
+    print('Updating QCoDeS from master...\n')
+    try:
+        subprocess.run(f'pip install -e {_source}', shell=True)
+        print(f"QCoDeS version {qcodes.__version__} is succesfully installed.")
+        sys.exit()
+    except ImportError:
+        warnings.warn("An unknown issue occured during update.\nThe changes shall be rolled back.", UserWarning, 2)
+        subprocess.run('conda deactivate && conda remove --name qcodes --all', shell=True)
+
+        print("Cloning QCoDeS from back up...\n")
+        subprocess.run(f'conda create --name {env_name} --clone {env_backup_name}', shell=True)
+        subprocess.run(f'conda remove --name {env_backup_name} --all', shell=True)
+
+        # Roll back to the roots
+        shutil.rmtree(_source)
+        shutil.copytree(_destination, _source, symlinks=True, ignore=None)
+        subprocess.run(f'activate qcodes && pip uninstall qcodes && pip install -e {_source}', shell=True)
+        shutil.rmtree(_destination)
+
+        sys.exit()
```

Comment:
as above

Reply:
I agree, this is unnecessary and the line will be removed.